### ISSUE TITLE
when reproducing loss gradually goes to nan

### ISSUE THREAD

**Issue by mkbzas777** [Jiaxuan-Li/EVCap#1]
Hi, I'm trying to reproduce your code and when I do the training, the loss gradually goes to nan. Is this correct?

**Comment by Jiaxuan-Li** [comment]
Hi, could you show more details on your training setup?
In our experiments, we use [vicuna-13b-v1.3](https://huggingface.co/lmsys/vicuna-13b-v1.3) (based on LLaMA1) as the caption generator. The training hyperparameters may not support the latest vicuna models

**Comment by mkbzas777** [comment]
Thank you for your reply.
I couldn't use the 13b version because my GPU didn't have enough memory to make it work, I tried swapping it out for the 7b version, maybe I need to adjust the parameters in a follow up lol.

**Comment by gpxhyfzqz** [comment]
Hi, I had the same problem, can you tell me how you handled it? Thank you very much!
![image](https://github.com/user-attachments/assets/6179b6a4-567e-4d8c-a41f-ba229e738b3c)
I'm training in RTX 4090, and the VLM version is vicuna-7b-v1.5.
